docs/user/exercises/modeling.rst

```diff
@@ -1,2 +1,193 @@
+
+Following screenshot illustrates the first section of the form. It consists of:
+
+- *Title*: Title of an exercise
+- *Categories*: Category of an exercise
+- *Difficulty*: Difficulty of an exercise. (No level, Easy, Medium or Hard)
+- *Mode*: Solving mode of an exercise. *This cannot be changed afterwards* (Individual or Team)
+- *Release Date*: Date after which students can access the exercise
+- *Due Date*: Date till when students can access the exercise
+- *Assessment Due Date*: Date till when instructors can assess the exercise submitted by students
+- *Inclusion in course score calculation*: Option that determines whether or not to include exercise in course score calculation
+- *Points*: Total point of an exercise
+- *Bonus Points*: Bonus points of an exercise
+- *Diagram Type*: Type of diagram that is used throughout an exercise
+
+    .. figure:: modeling/create-modeling-exercise-form-1.png
+              :align: center
+
+.. note::
+   Fields marked with red are mandatory to be filled.
+
+.. note::
+   - Field **Diagram Type** determines the components that students/instructors can use while working on the exercise.
+   - For example: If the instructor selects Class diagram as its Diagram Type, users (instructors/students) will now only be able to use components of class diagram throughout the exercise.
+
+    .. figure:: modeling/class-diagram-diagram-type.png
+              :align: center
+
+Following screenshot illustrates the second section of the form. It consists of:
+
+- *Enable automatic assessment suggestions*: Option that determines whether or not to enable assessment suggestions automatically
+- *Problem Statement*: Field to state problem statement of an exercise
+- *Assessment Instructions*: Instructions for instructor while assessing the submission
+
+    .. figure:: modeling/create-modeling-exercise-form-2.png
+              :align: center
+
+.. note::
+    If you are not clear about any of the fields, you can access to the field hints by hovering on |hint| icon, when available.
+
+    .. figure:: modeling/create-modeling-exercise-form-hint.png
+              :align: center
+
+Following screenshot illustrates the last section of the form. It consists of:
+
+- *Example Solution*: Example solution of an exercise
+- *Example Solution Explanation*: Explanation of example solution of an exercise
+- *Example Solution Publication Date*: Date after which example solution is accessible for students
+
+    .. figure:: modeling/create-modeling-exercise-form-3.png
+              :align: center
+
+
+Once you are done defining the schema of an exercise, you can now create an exercise by clicking on |save| button.
+You will then be redirected to **Example Submissions for Assessment Training** Page
+
+    .. figure:: modeling/example-submission-for-assessment-training.png
+              :align: center
+
+In this page, you can either *Create Example Submission* or *Use as Example Submission* for Assessment Training.
```

Comment:
```suggestion
In this page, you can either *Create Example Submission* or *Use as Example Submission* for Assessment Training.
Example submission can be used to assess the submissions of students semi-automatically.
Artemis uses those submissions to automatically apply the known assessment comments to similar model elements in other submissions as well.
```
I suggest moving the explanation what this page is used for to a place before the description of the possible interactions on the page.